salt/modules/debian_ip.py

```diff
@@ -899,17 +899,18 @@ def _parse_settings_bond_1(opts, iface, bond_def):
             _log_default_iface(iface, binding, bond_def[binding])
             bond.update({binding: bond_def[binding]})
 
-    if 'use_carrier' in opts:
-        if opts['use_carrier'] in _CONFIG_TRUE:
-            bond.update({'use_carrier': '1'})
-        elif opts['use_carrier'] in _CONFIG_FALSE:
-            bond.update({'use_carrier': '0'})
+    if __grains__['os'] == "Ubuntu" and __grains__['osrelease_info'][0] >= 16:
```

Comment:
It should not use `use_carrier` for Ubuntu equal or greater than 16.04, because this option is there unknown.